arch/x86_64/dynamic.S

```diff
@@ -42,6 +42,7 @@
 
 GLOBAL(__dentry__)
 	.cfi_startproc
+	pushf
```

Comment:
I'm afraid of 16-byte stack alignment

Reply:
Can you please share more details about your worry.

Reply:
x86 ABI requires 16 byte alignment for the function entry.  So `mcount` and `__dentry__` in libmcount take care of the alignment when saving registers.

Reply:
I thought that the line dynamic.S:67 (andq $0xfffffffffffffff0, %rsp) is taking care of that.

Reply:
Oh, you're right.  This part was changed several times so I forgot the history, sorry! :)